orchestration/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,56 @@
+{{- with .Values.serviceAccount }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .k8sName }}
+  annotations:
+    iam.gke.io/gcp-service-account: {{ .googleName | quote }}
+---
+{{- $pspName := printf "%s-encode-psp" .k8sName }}
+apiVersion: policy/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: {{ $pspName }}
+spec:
+  privileged: false
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  fsGroup:
+    rule: RunAsAny
+  volumes:
+    - '*'
+---
+# Permissions needed by a k8s SA to run the ClinVar workflow.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ .k8sName }}-role
+rules:
```

Comment:
are these rules intentionally different than what was in the deleted role file?

Reply:
Yes, the role file was over-provisioning permissions (I think we copy-pasted it from the Argo quickstart)